README.it.md

```diff
@@ -0,0 +1,183 @@
+<br>
+Il launcher è scritto in TypeScript (Electron) and Python, che gestisce il sistema di torrenting appoggiandosi a libtorrent.
+
+## Caratteristiche
+
+- Motore di ricerca automatizzato sulle fonti di repack dal [Megathread]("https://www.reddit.com/r/Piracy/wiki/megathread/")
+- Client Bittorrent integrato
+- Integrazione How Long To Beat (HLTB) nella pagina del gioco
+- Percorso del download Personalizzato
+- Notifiche di aggiornamenti sulla list dei repacks
+- Supporto Windows e Linux
+- Costantemente Aggiornato
+- E molto altro ...
+
+## Installazione
+
+Segui i seguenti passi:
+
+1. Scarica l'ultima versione di Hydra dalla pagina [Releases](https://github.com/hydralauncher/hydra/releases/latest).
+   - Scarica solo il file .exe per installare Hydra su Windows.
+   - Scarica il file .deb o .rpm o .zip per Linux. (Dipende dalla tua distro Linux)
+2. Esegui il file scaricato.
+3. Goditi Hydra!
+
+## <a name="contribuire"> Contribuire
+
+### <a name="unisciti-su-telegram"></a> Unisciti su Telegram
+
+Puoi unirti alle nostre conversazioni sul canale [Telegram](https://t.me/hydralauncher).
+
+### Forka e Clona la repository
+
+1. Forka la repository [(clicca qui per forkare)](https://github.com/hydralauncher/hydra/fork)
+2. Clona il tuo codice forkato `git clone https://github.com/your_username/hydra`
+3. Crea un nuovo branch
+4. Aggiungi le modifiche (push)
+5. Invia la richiesta di pull
+
+### Modi in cui contribuire
+
+- Traduzione: Vogliamo rendere Hydra disponibile a più persone possibile. Sentiti libero di tradurre in altre lingue o aggiornare e migliorare quelle già disponibili su Hydra.
+- Programmazione: Hydra è programmato in TypeScript, Electron e un po' di Python. Se intendi contribuire unisciti al nostro [Telegram](https://t.me/hydralauncher)!
+
+### Struttura del Progetto
+
+- client-torrent: Usiamo libtorrent, una libreria Python, per gestire i download dei torrent
+- src/renderer: l'UI dell'applicazione
+- src/main: tutta la logica qui.
+
+## Compilazione
+
+### Installa Node.js
+
+Assicurati di avere Node.js installato sulla tua macchina. Scaricalo e installalo da [nodejs.org](https://nodejs.org/).
+
+### Installa Yarn
+
+Yarn è un gestore di pacchetti per Node.js. Se non hai ancora installato Yarn segui le istruzioni su [yarnpkg.com](https://classic.yarnpkg.com/lang/en/docs/install/).
+
+### Installa le dipendenze Node
+
+Naviga alla cartella del progetto e installa le dipendenze Node con Yarn:
+
+```bash
+cd hydra
+yarn
+```
+
+### Installa Python 3.9
+
+Assicurati di avere Python 3.9 installato. Puoi scaricarlo da [python.org](https://www.python.org/downloads/release/python-3913/).
+
+### Installa le Dipendenze Python
+
+Installa le dipendenze con pip:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Variabili d'ambiente
+
+Avrai bisogno di una chiave API SteamGridDB per poter caricare le icone di gioco.
+Se intendi avere onlinefix come repacker dovrai aggiungere le tue credenziali al file .env
+
```

Comment:
wtf is this environment variables?

please only consider the main [README](https://github.com/hydralauncher/hydra/blob/main/README.md).